bpf/bpf_host.c

```
@@ -1271,9 +1275,11 @@ int cil_from_netdev(struct __ctx_buff *ctx)
 	if (ctx->mark == MARK_MAGIC_DECRYPT)
 		return CTX_ACT_OK;
 #endif
+	ret = tcx_early_hook(ctx, proto);
+	if (ret != CTX_ACT_OK)
+		return ret;
```

@borkmann please wire up error handling and exit via `drop_err` here. Let's not silently drop packets.

---

@borkmann @aspsk anyone?

---

yep, will push a pr shortly